bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/container.jsx

```diff
@@ -17,6 +17,7 @@ export default withTracker(({ userId }) => {
     user: mapUser(Users.findOne({ userId })),
     userInBreakout: !!findUserInBreakout,
     breakoutSequence,
+    currentUser: Users.findOne({ userId: Auth.userID }),
```

Comment:
This is an unprotected find and will re-render every time the local user changes. It's just pushing the re-render problem from the parent component into the children rather than removing the problem.